spec.html

```diff
@@ -21754,8 +21762,8 @@ <h1>InnerModuleInstantiation ( _module_, _stack_, _index_ )</h1>
               1. Set _module_.[[DFSAncestorIndex]] to _index_.
               1. Increase _index_ by 1.
               1. Append _module_ to _stack_.
-              1. For each String _required_ that is an element of _module_.[[RequestedModules]], do
-                1. Let _requiredModule_ be ? HostResolveImportedModule(_module_, _required_).
+              1. Let _requiredModules_ be _module_.GetRequestedModules()
+              1. For each module _requiredModule_ that is an element of _requiredModules_, do
```

Comment:
This suggestion mirrors the language used here: https://tc39.github.io/ecma262/#sec-enumerableownpropertynames
```suggestion
              1. For each element _requiredModule_ of _requiredModules_ in List order, do
```

Not sure if we have a prevailing convention on this or not.

Reply:
Fixed.  Thanks!